rpcd/playbooks/roles/rpc_pre_upgrade/tasks/gather_stats.yml

```diff
@@ -44,6 +44,13 @@
   register: neutron_agentlist_result
   delegate_to: "{{ groups['utility_all'][0] }}"
 
+- name: Gather orchestration service list
+  shell: "source ~/openrc; openstack orchestration service list --insecure"
```

Comment:
openstack should use the clouds.yaml so no need to source the openrc if I'm not mistaken.

You can then move to use the command module.